src/War3Map.Template.Launcher/War3Map.Template.Launcher.csproj

```diff
@@ -2,16 +2,16 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\War3Map.Template.Common\War3Map.Template.Common.csproj" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="War3Net.Build" Version="1.3.3" />
-    <PackageReference Include="War3Net.CSharpLua.CoreSystem" Version="1.2.3" />
+    <PackageReference Include="War3Net.Build" Version="1.7.0" />
+    <PackageReference Include="War3Net.CSharpLua.CoreSystem" Version="1.7.3" />
```

Comment:
War3Net.CSharpLua and War3Net.CSharpLua.CoreSystem should stay in sync. Since I haven't updated War3Net.Build in a while, it still uses CSharpLua 1.7.0, instead of the latest 1.7.3, so you should add:
```
<PackageReference Include="War3Net.CSharpLua" Version="1.7.3" />
```